datacenter/dtr/2.2/guides/admin/configure/garbage-collection.md

```diff
@@ -0,0 +1,97 @@
+---
+description: Configure garbage collection in Docker Trusted Registry
+keyworkds: docker, registry, garbage collection, gc, space, disk space
+title: Docker Trusted Registry 2.2 Garbage Collection
+---
+
+## TL;DR
+
+1. Garbage Collection (GC) reclaims disk space from your storage by deleting
+unused layers
+2. GC can eb configured to run automatically with a cron schedule, and can also
```

Comment:
* eb -> be
* Use hard stops instead of ";". It's easier for scanning the sentence with your eyes.
* No need to use "GC", use "garbage collection". Don't capitalize it, since it's not a product name